docs/visual-basic/whats-new/index.md

```diff
@@ -2,14 +2,12 @@
 description: "Learn more about: What's new for Visual Basic"
 title: What's new
 ms.date: 10/24/2018
-ms.update-cycle: 365-days
```

Comment:
Let's leave this as 365. I should make a review once per year to address any potential updates for VB code to consume new C# features.